fake_fire/init.lua

```diff
@@ -326,10 +329,22 @@ minetest.register_alias("fake_fire:flint", "fake_fire:flint_and_steel")
 minetest.register_lbm({
 	name = "fake_fire:reload_particles",
 	label = "restart fire particles on reload",
-	nodenames = {"fake_fire:fancy_fire"},
+	nodenames = "fake_fire:fancy_fire",
```

Comment:
fyi: the docs specify a table here (like before the change) https://github.com/minetest/minetest/blob/b89eb605b7020998fbf5d7142033ceb5275b3f71/doc/lua_api.txt#L7930

Reply:
oh yes, that got lost when I tried to put everything into one lbm - fixed